ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -84,11 +84,18 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
     transition,
   ) {
     await this.getAllTargets(transition);
+    const projects = this.modelFor('scopes.scope.projects');
+    const projectIds = projects.map((project) => project.id);
 
     const filters = { scope_id: [], id: { values: [] }, type: [] };
     scopes.forEach((scope) => {
       filters.scope_id.push({ equals: scope });
     });
+    if (scopes.length === 0) {
```

Comment:
One issue is this assumes that the client daemon is up...which I think is fine for now

Reply:
In one of your PRs you add the `isClientDaemonRunning`. I could utilize that here to bypass this step.

Reply:
Sure but you'd need to manually filter it down then